setup.cfg

```diff
@@ -26,18 +26,18 @@ description = Software development kit for open source D-Wave tools
 
 [options]
 install_requires =
-    dimod==0.12.18
-    dwave-cloud-client==0.13.3
+    dimod==0.12.20
+    dwave-cloud-client==0.13.4
     dwave-gate==0.3.3
-    dwave-hybrid==0.6.13
-    dwave-inspector==0.5.2
-    dwave-networkx==0.8.16
-    dwave-optimization==0.5.1
-    dwave-preprocessing==0.6.7
-    dwave-samplers==1.4.0
-    dwave-system==1.29.0
+    dwave-hybrid==0.6.14
+    dwave-inspector==0.5.3
+    dwave-networkx==0.8.17
+    dwave-optimization==0.6.0
```

Comment:
The submodule we're actually using is a little ahead of the released version: it includes [this commit](https://github.com/dwavesystems/dwave-optimization/commit/dfebf27841af883057d6ec958426239551302a98)  
I think that's probably okay but wanted to raise it here.